certora/helpers/MetaMorphoHarness.sol

```diff
@@ -44,4 +46,43 @@ contract MetaMorphoHarness is MetaMorpho {
     function maxFee() external pure returns (uint256) {
         return ConstantsLib.MAX_FEE;
     }
+
+    function nextGuardianUpdateTime() external view returns (uint256 nextTime) {
+        nextTime = block.timestamp + timelock;
+
+        if (pendingTimelock.validAt != 0) {
+            nextTime = Math.min(nextTime, pendingTimelock.validAt + pendingTimelock.value);
+        }
+
+        uint256 validAt = pendingGuardian.validAt;
+        if (validAt != 0) nextTime = Math.min(nextTime, validAt);
```

Comment:
I'm not sure to understand, line 56 `nextTime` is not defined and I don't think it works for the else branch either. For example, assuming `pendingGuardian.validAt == 0`, `timelock = 1 year`, `pendingTimelock.validAt = tomorrow`, `pendingTimelock.value = 1 days`, then the formula gives a result in one year from now, but using the pending timelock we would get it 2 days from now (because `timelock < pendingTimelock.validAt + pendingtimelock.value`)

Reply:
>line 56 nextTime is not defined

indeed, edited

>For example, assuming pendingGuardian.validAt == 0, timelock = 1 year, pendingTimelock.validAt = tomorrow, pendingTimelock.value = 1 days, then the formula gives a result in one year from now, but using the pending timelock we would get it 2 days from now (because timelock < pendingTimelock.validAt + pendingtimelock.value)

indeed...

Reply:
I prefer the current version now, it uses `pendingTimelock.validAt` directly without the `zeroFloorSub`. I like that you see the 3 paths too:
- either use the `timelock` value now to submit a proposal (& accept it)
- or if there is a pending timelock, at `pendingTimelock.validAt` use the new `pendingTimelock.value` to submit a proposal (& accept it)
- or if there is a pending guardian, wait to accept that proposal at `validAt`

Reply:
Ok, as you prefer